Training6/training-6-sagas-login/src/sagas/index.js

```diff
@@ -0,0 +1,36 @@
+import axios from "axios";
+import { call, put, takeLatest } from "redux-saga/effects";
+
+function* workerLogin({ values }) {
+  try {
+    const url = "http://localhost:9000/login";
+    const response = yield call(axios.post, url, values);
+    if (!response.data.token) {
+      yield put({ type: "LOGIN_ERROR", error: response.data.error });
+    } else {
+      yield put({ type: "LOGIN_SUCCESS", token: response.data.token });
+    }
+  } catch (error) {
+    yield put({ type: "LOGIN_ERROR", error });
+  }
+}
+
+function* workerGetUser({ token }) {
```

Comment:
we don't need pass token into action payload, we need using `yield select(selector to get token)`